apps/omni/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "esm": true,
+  "compilerOptions": {
+    "rootDirs": ["src"],
+    "outDir": "dist",
+    "lib": ["es2020"],
+    "module": "CommonJS",
+    "target": "es2020",
+    "moduleResolution": "node",
+    "esModuleInterop": true,
+    "types": ["node"]
```

Comment:
enable strict mode

Reply:
I am lazy tho...

Aight I will do it later